src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/prometheus/active-alert-list/active-alert-list.component.ts

```diff
@@ -105,6 +108,20 @@ export class ActiveAlertListComponent extends PrometheusListHelper implements On
         cellTemplate: this.externalLinkTpl
       }
     ];
+    this.prometheusService.getBalancerStatus().subscribe((resp) => {
+      this.status = resp['optimize_result'];
+      this.active = resp['active'];
+    });
+
+    if (
+      this.status ===
+        'Unable to find further optimization, or pool(s) pg_num is decreasing, or distribution is already perfect' &&
```

Comment:
not an ideal condition.. if the text got changed in the balancer status result, then we'll need to modify here as well. Are you sure we can't use anything else?